### ISSUE TITLE
Using Densenet with  MultiLabelMarginCriterion()

### ISSUE THREAD

**Issue by Mahmood1968** [dell-ai-engineering/BigDL-ImageProcessing-Examples#10]
I have used densent with  MultiLabelMarginCriterion() and learning rate =0.0005 with batch size =64 and epochs =5 . Here is the loss function and learning
![desnsenet](https://user-images.githubusercontent.com/24418838/47237470-ca7bac00-d3a4-11e8-8295-67c57ba2cfd6.png)
 rate . please look at it

**Comment by Mahmood1968** [comment]
<img width="405" alt="loss_lr" src="https://user-images.githubusercontent.com/24418838/47315442-36128300-d60a-11e8-853e-2267a86dc680.png">
here using Jupyter notebook , loss function and learning rate

**Comment by leela-uppuluri** [comment]
@Mahmood1968 
I think we can close this issue as this model works with different loss function "BinaryCrossEntropy"
